packages/protoc-gen-connect-query-react/.eslintignore

```diff
@@ -1 +1,2 @@
-dist
\ No newline at end of file
+dist
+snapshots/**/*.d.ts
```

Comment:
Is it necessary? There is an eslint-disable comment in every file.

Reply:
Yeah this is a little confusing for me as well. Eslint was complaining about processing any file that the tsconfig file didn't process. It only happens for `d.ts` files so maybe I need to adjust the tsconfig file somehow to process d.ts files specifically... I'll dig in a little more.

Reply:
It's weird, it's basically because eslint tries to provide the d.ts files to be compiled and our tsconfig doesn't seem to include these files for... reasons? I've made modifications to skip type checking for d.ts files and instead just lint for everything else. Should be ok.